.phpstorm.meta.php

```diff
@@ -7,67 +7,68 @@
 
 		// TODO: I'd like to use Plugin::SERVICES directly here but it doesn't seem to work.
 		map( [
```

Comment:
Isn't the arrow alignment incorrect now?

Reply:
Fixed